src/time_axis_scale.rs

```diff
@@ -53,15 +53,15 @@ impl TimeAxisScale {
     }
 }
 
-impl AxisScale for TimeAxisScale {
-    fn ticks(&self) -> Vec<AxisTick> {
+impl Scale for TimeScale {
+    fn ticks(&self) -> Vec<Tick> {
         let scale = self.clone();
         ((self.time_from)..self.time_to + 1)
             .into_iter()
             .step_by(scale.step as usize)
             .map(move |i| {
                 let location = (i - scale.time_from) as f32 * scale.scale;
-                AxisTick {
+                Tick {
                     location: NormalisedValue(location),
                     label: self.labeller.as_ref().map(|l| (l)(i)),
```

Comment:
Could you use `ref_map` here?

Reply:
Nevermind, I just realised `ref_map` is a separate crate...

Reply:
`ref_map` doesn't appear to be available for `Option`, or anywhere else... Are you referring to the ref-map crate?